### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by hokyungh**
*Issue #, if available:*

*Description of changes:*
Update documentation for android.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by zhinang-amazon — APPROVED** [review]

**Review by dylonChime — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -70,7 +70,46 @@ The following camera and microphone permissions need to be granted to enable aud
 
 More details on how to request permissions can be found in [Android](https://developer.android.com/training/permissions/requesting) and [iOS](https://developer.apple.com/documentation/avfoundation/cameras_and_media_capture/requesting_authorization_for_media_capture_on_ios?language=objc) official documents.
 
-## 3. Connect React Native code to the Amazon Chime SDK
+## 3. Add libraries for SDK
+
+### Android
```
Comment (linsang21) [inline README.md:75]: Only android?

File: README.md [hunk 2]
```diff
@@ -70,7 +70,46 @@ The following camera and microphone permissions need to be granted to enable aud
...
+```
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+dependencies {
+    implementation(name: 'amazon-chime-sdk', ext: 'aar')
+    implementation(name: 'amazon-chime-sdk-media', ext: 'aar')
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.3'
+    implementation 'com.google.code.gson:gson:2.8.6'
```
Comment (linsang21) [inline README.md:108]: Why do we require these three?

Comment (hokyungh) [reply README.md:108]: Our framework doesn't seem to contain these libraries by default, so need to add it.

File: README.md (hunk 1 above)
Comment (hokyungh) [reply README.md:75]: iOS doesn't seem to be have extra dependencies

**Review by linsang21 — APPROVED** [review]

**Pull request merged**